packages/sw-build/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "sw-build",
+  "version": "0.0.8",
+  "description": "A CLI tool to generate a service worker and a file manifest making use of the sw-lib module.",
+  "keywords": [
+    "sw-fw",
+    "service worker",
+    "caching",
+    "fetch requests",
+    "offline",
+    "file manifest"
+  ],
+  "main": "build/index.js",
+  "files": [
+    "src",
+    "build"
+  ],
+  "engines": {
+    "node": ">=4.0.0"
+  },
+  "author": "Google's Web DevRel Team",
+  "license": "Apache-2.0",
+  "repository": "googlechrome/sw-helpers",
+  "bugs": "https://github.com/googlechrome/sw-helpers/issues",
+  "homepage": "https://github.com/GoogleChrome/sw-helpers/tree/master/packages/sw-build",
+  "dependencies": {
+    "sw-lib": "^0.0.8"
```

Comment:
Double-checking that this is a legitimate runtime dependency that needs to be installed alongside `sw-build`.